languagetool-rpm-package/src/main/resources/README

```diff
@@ -4,9 +4,9 @@ LanguageTool RPM Module
 Introduction
 ------------
 
-This is a short description LanguageTool RPM module. It primary purpose is to
-generate LanguageTool RPM package. The package is primarily aimed at RPM-based
-Linux distributions. The main goal is to structure LanguageTool so that developers
+This is a short description Language Tool RPM module. It primary purpose is to
```

Comment:
Looks like the wrong spelling has been re-introduced here.

Reply:
Will fix.